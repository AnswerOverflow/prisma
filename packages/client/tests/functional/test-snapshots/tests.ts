import testMatrix from './_matrix'

// @ts-ignore this is just for type checks
declare let prisma: import('@prisma/client').PrismaClient

testMatrix.setupTestSuite(
  (suiteConfig, suiteMeta) => {
    test('example', () => {
      expect(suiteConfig.provider + '!').toMatchSnapshot()
    })
  },
  {
    optOut: {
      from: ['mysql', 'cockroachdb', 'sqlserver'],
      reason: 'just because',
    },
  },
)
