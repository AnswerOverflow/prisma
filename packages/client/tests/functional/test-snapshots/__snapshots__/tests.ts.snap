// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test-snapshots (provider=mongodb) example 1`] = `mongodb!!!`;

exports[`test-snapshots (provider=postgresql) example 1`] = `postgresql!!!`;

exports[`test-snapshots (provider=sqlite) example 1`] = `sqlite!!!`;
